lib/matplotlib/axis.py

```diff
@@ -829,13 +829,14 @@ def _get_autoscale_on(self):
     def _set_autoscale_on(self, b):
         """
         Set whether this Axis is autoscaled when drawing or by
-        `.Axes.autoscale_view`.
+        `.Axes.autoscale_view`. If b is None, then the value is not changed.
```

Comment:
```suggestion
        `.Axes.autoscale_view`.
        
        If *b* is None, then the value is not changed.
```
private method, but still...

Reply:
Yeah this is a good catch, this gets wrapped by public methods so I reverted the change.